### ISSUE TITLE
FindBugs' Nullcheck fails in SourceImpl.java

### ISSUE THREAD

**Issue by fniephaus** [oracle/graal#978]
Not sure why I see this error now, the [lines of interest](https://github.com/oracle/graal/blob/a237181bb5efe0f6bc1eeeec42e3b4b012abe8c9/truffle/src/com.oracle.truffle.api/src/com/oracle/truffle/api/source/SourceImpl.java#L275-L284) haven't changed in a while and so hasn't the version of FindBugs that mx uses. Maybe because of a different FingBugs-related change in mx?

Error message:
```
M C RCN: Nullcheck of other at line 281 of value previously dereferenced in com.oracle.truffle.api.source.SourceImpl$Key.compareCharacters(CharSequence, CharSequence)  At SourceImpl.java:[line 278]
```

**Comment by fniephaus** [comment]
Ok, here we go: https://github.com/graalvm/mx/commit/c3cfe9980ecfdc0a9a858122f8a1bc04c2ec72ab (mx upgrade from `5.210.1` to `5.210.2`).

**Comment by dougxc** [comment]
A fix to mx is on it's way. I've [highlighted](https://github.com/graalvm/mx/commit/48698eee3e9ad18e3a2f03511ffc4112c788dc3b#r32318770) the problem.

**Comment by fniephaus** [comment]
Fixed in mx `5.210.3` via https://github.com/graalvm/mx/commit/302aac4afea7b0e1ef19201c848d77ac4adb527d.
